spartan/aztec-network/values.yaml

```diff
@@ -34,6 +34,7 @@ aztec:
   epochDuration: 16 # how many L2 slots in an epoch
   epochProofClaimWindow: 13 # in L2 slots
   realProofs: false
+  l1DeploymentMnemonic: "test test test test test test test test test test test junk" # the mnemonic used when deploying contracts
```

Comment:
this isn't just used when deploying contracts, right? This is the eth node's mnemonic?

Reply:
Correct, but whatever is set in this field is what will be used for contract deployments.